README.md

```diff
@@ -5,7 +5,7 @@ Write down two truths and one lie below.
 
 Once you're done, submit a pull request (do NOT commit directly to the main branch)! We'll try to guess the lie before we merge your branch with the main branch.
 
-YOUR NAME HERE:
-1. A truth (or a lie)
-2. A lie (or a truth)
-3. Another truth (or your lie)
+Meredith: (ideas thanks to Alice)
+1. I love sifakas.
+2. I love brown lemurs.
+3. I love ring-tailed lemurs.
```

Comment:
Ring-tailed lemurs are awesome! But this is too reasonable of a take to be the truth. This is the lie.

Reply:
Meredith you hate Ring Tails

Reply:
TRUTH^@christofer